packages/virtual-core/src/index.ts

```diff
@@ -133,6 +133,7 @@ const supportsScrollend =
 export const observeElementOffset = <T extends Element>(
   instance: Virtualizer<T, any>,
   cb: (offset: number, isScrolling: boolean) => void,
+  debounceFallbackEnabled = false,
```

Comment:
Don't need to pass it here, we can read it from `instance.options.scrollendEventEnabled`